src/features/unmarkedWsNoticeList/UnmarkedWsNoticeList.tsx

```diff
@@ -99,6 +110,32 @@ const UnmarkedWsNoticeList = ({
             <UnmarkedWsNoticeDetails {...row.original} />
           </Grid>
         )}
+        renderTableToolsTop={({ selectedRows }, { resetSelectedRows }) => {
+          const offers = getDraftedOffers(selectedRows);
```

Comment:
Should this logic be in `ApplicationListTools` instead? Now it's still possible to send the BE request even though `offers` contains ones without places?